lms/static/sass/shared-v2/_components.scss

```diff
@@ -148,7 +148,7 @@
   @include box-sizing(border-box);
 
   margin: 0 auto;
-  padding: ($baseline*0.75) ($baseline*2);
+  padding: ($baseline*0.75) 20px;
```

Comment:
?

Reply:
yea we need to align all the banners with the laft margin of the body content....per request of the design team.